openapi.yaml

```diff
@@ -3291,6 +3291,54 @@ paths:
                       `~/.ssh/authorized_keys` file automatically.
                     items:
                       type: string
+                  interfaces:
+                    type: object
+                    description: >
+                      The Network Interface(s) to add to this Linode. This field is only required when creating
+                      a Linode that is attached to a Virtual Local Area Network (VLAN). **Note**: You must create
+                      the VLAN prior to attaching a Linode to it.
+
+                      * You can add up to 3 Network Interfaces, `eth0` - `eth2`.
+
+                      * A Linode can only have one Network Interface of `type: default`. This is the Linode's
+                      public Network Interface, which allows the Linode to communicate across public networks.
+
```

Comment:
* If only `type: additional` interfaces are specified, then this Linode won't have direct access to public Internet. This way you can create truly private Linodes. You can add public interface to the default or a custom boot config later.